tests/xtro-sharpie/macOS-StoreKit.ignore

```diff
@@ -1,7 +1,3 @@
 # Fixed in XAMCORE_4_0
 !unknown-native-enum! SKCloudServiceAuthorizationStatus bound
 !unknown-native-enum! SKCloudServiceCapability bound
-
-# Breaking change by Apple in Xcode 10, so we don't follow suit.
-# https://trello.com/c/A34S0kLY/125-41782055-skproductdownloadable-crashes-on-macos-1011-when-compiled-with-xcode-10
```

Comment:
Minor but I would have also updated the binding file: https://github.com/xamarin/xamarin-macios/pull/4528/files#diff-e9aebf3ee5f3c844607eb2da9d2feb14L218 (I updated the Trello card).